lib/susedistribution.pm

```diff
@@ -673,7 +673,6 @@ sub activate_console {
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
```

Comment:
and this comment about a bug , it relates to assert_screen ? confusing is that it saying something about "need to wait"

Reply:
Probably related to `assert_screen` indeed.